src/composables/useUrls.ts

```diff
@@ -0,0 +1,12 @@
+import { configService } from '@/services/config/config.service';
+
+export default function useUrls() {
+  function resolve(uri: string): string {
+    if (!uri) return '';
+    return uri
+      .replace('ipfs://', `https://${configService.env.IPFS_NODE}/ipfs/`)
+      .replace('ipns://', `https://${configService.env.IPFS_NODE}/ipns/`);
```

Comment:
I think in this instance `resolve` can live outside the composable right? just to reduce the number of times we redeclare this function.

Reply:
Yup good point.